### PR TITLE
Added new Documentation section

### PR DESCRIPTION
**Pull request by LenzGr**
Added new pages that provide some more detailed information on how to contribute to our documentation.

Signed-off-by: Lenz Grimmer <lenz.grimmer@percona.com>

### CODE REVIEW

**Comment by LenzGr** [comment]
I'm still struggling with getting `contribute/documentation/_index.md` rendering properly. Currently, the first headline and the bullet points are not rendered. Any help on getting this fixed would be appreciated.

**Review by dbazhenov — APPROVED** [review]
Thanks

File: content/contribute/documentation/locations.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+| [Percona Server for MySQL](https://www.percona.com/doc/percona-server/LATEST/index.html)  | https://github.com/percona/percona-server/tree/master/doc |
+| [Percona XtraDB Cluster](https://www.percona.com/doc/percona-xtradb-cluster/LATEST/index.html) | https://github.com/percona/percona-xtradb-cluster/tree/master/doc/ |
+| [Percona XtraBackup](https://www.percona.com/doc/percona-xtrabackup/LATEST/index.html) | https://github.com/percona/percona-xtrabackup//tree/master/storage/innobase/xtrabackup/doc/ |
+| [Percona Server for MongoDB](https://docs.percona.com/percona-server-for-mongodb/latest/index.html) | https://github.com/percona/psmdb-docs |
+| [Percona Backup for MongoDB](https://www.percona.com/doc/percona-backup-mongodb/index.html) | https://github.com/percona/percona-backup-mongodb |
+| [Percona Toolkit](https://www.percona.com/doc/percona-toolkit/LATEST/index.html) | https://github.com/percona/percona-toolkit/tree/master/docs/ |
+| [Percona Software Repositories Documentation](https://www.percona.com/doc/percona-repo-config/index.html) | https://github.com/percona/repo-config-docs |
```
Comment (patrickbirch) [inline content/contribute/documentation/locations.md:33]: [ProxySQL Admin Tool] (https://github.com/percona/proxysql-admin-tool-doc/source) |https://github.com/percona/proxysql-admin-tool-doc/tree/main/source|

File: content/contribute/documentation/locations.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+| [Percona Distribution for MySQL](https://www.percona.com/doc/percona-distribution-mysql/LATEST/index.html) | https://github.com/percona/pdmysql-docs |
+| [Percona Distribution for PostgeSQL](https://www.percona.com/doc/postgresql/LATEST/index.html) | https://github.com/percona/postgresql-docs |
+| [Percona Distribution for MySQL Operator](https://www.percona.com/doc/kubernetes-operator-for-pxc/index.html) | https://github.com/percona/percona-xtradb-cluster-operator/tree/pxc-docs |
+| [Percona Distribution for MongoDB Operator](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/index.html) | https://github.com/percona/percona-server-mongodb-operator/tree/K8s-PSMDB-docs |
+| [Percona Distribution for PostgreSQL Operator](https://www.percona.com/doc/kubernetes-operator-for-postgresql/index.html) | https://github.com/percona/percona-postgresql-operator/tree/pgo-docs |
+| [Percona Server for MySQL](https://www.percona.com/doc/percona-server/LATEST/index.html)  | https://github.com/percona/percona-server/tree/master/doc |
```
Comment (patrickbirch) [inline content/contribute/documentation/locations.md:27]: Should we send them to the source directory instead of doc?

Comment (LenzGr) [reply content/contribute/documentation/locations.md:27]: I was trying to point right to the actual location of the documentation, not the root of the repository. Do you think that's too confusing?

File: content/contribute/documentation/_index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+submit the concrete changes yourself! It's really easy but can be intimidating
+the first time. This guide should take the edge off and you'll see your changes
+live in a matter of hours!
+
+* Similar to our software, our documentation is open source and is managed using
+  the same tools and processes that we use to incorporate changes to our code
+  base.
+* Several of our manuals already provide an "Edit this page" link that you can
+  use to perform the actual documentation changes that you would like to see in
+  your web browser. Note that this process requires having an account on github
```
Comment (nastena1606) [inline content/contribute/documentation/_index.md:34]:
```suggestion
  your web browser. Note that this process requires having an account on GitHub
```

File: content/contribute/documentation/locations.md (hunk 1 above)
Comment (LenzGr) [reply content/contribute/documentation/locations.md:33]: Added, thanks!

File: content/contribute/documentation/_index.md (hunk 3 above)
Comment (LenzGr) [reply content/contribute/documentation/_index.md:34]: Fixed.

**Review by dbazhenov — APPROVED** [review]

**Pull request merged**